cflib/serialize/impl/serializeberimpl.h

```diff
@@ -457,32 +457,32 @@ inline void deserializeBER(QHash<Key, T> & cl, const quint8 * data, int len, BER
 }
 
 // ----------------------------------------------------------------------------
-// QMap
+// QMultiMap
```

Comment:
This was a `QMap` but used as a multi map. So I have just replaced it. Alternative we could provide a `QMap` and a `QMultiMap` implementation. Just not sure if this is desired.